packages/dds/tree/src/test/shared-tree-core/edit-manager/editManagerPerf.test.ts

```diff
@@ -201,37 +202,141 @@ export function testPerf() {
+				// When bunchCommits is true, it simulates the behavior of op bunching feature where
+				// the commits are processed by the EditManager in a together.
+				const bunchCommits = [true, false];
+				for (const bunchCommit of bunchCommits) {
+					// This test simulates the following inputs to the EditManager:
+					//   - Add trunk edit T1 with a ref seq# pointing to edit 0
+					//   ...(incrementing the ref seq# for each T)
+					//   - Add trunk edit Tc with a ref seq# pointing to edit Tc-1
+					//   => we start measuring from here
+					//   - Add peer edit P1 with a ref seq# pointing to edit 0
+					//   ...(not incrementing the ref seq# for each P)
+					//   - Add peer edit Pc with a ref seq# pointing to edit 0
+					// This defines the following relationships between edits:
+					//   (0)─(T1)─...─(Tc)
+					//     └───────────────(P1)─...─(Pc)
+					// Before we start measuring, the EditManager has the following structure:
+					//   (0)─(T1)─...─(Tc)                -> Trunk
+					//     └─                             -> Peer branch
+					// By the end of the test, the EditManager has the following structure:
+					//   (0)─(T1)─...─(Tc)─(P1)─...─(Pc)  -> Trunk
+					//     └───────────────(P1)─...─(Pc)  -> Peer branch
+					it(`Rebase ${P} peer commits over ${T} trunk commits ${bunchCommit ? "with op bunching" : ""}`, () => {
+						const rebaser = new NoOpChangeRebaser();
+						const manager = testChangeEditManagerFactory({ rebaser }).manager;
+						const run = rebasePeerEditsOverTrunkEdits(
+							P,
+							T,
+							manager,
+							() => TestChange.emptyChange,
+							true /* defer */,
+							bunchCommit,
+						);
+						rebaser.rebasedCount = 0;
+						rebaser.invertedCount = 0;
+						rebaser.composedCount = 0;
+						run();
+						const actual = {
+							rebased: rebaser.rebasedCount,
+							inverted: rebaser.invertedCount,
+							composed: rebaser.composedCount,
+						};
+						const expectedComposed = bunchCommit
+							? // With op bunching, all the peer commits are processed together. They are added
+								// to the peer branch and the peer branch is then merged with the trunk.
+								// As part of rebasing the new peer edits to the trip of the trunk, we compose...
+								//   - the trunk edits: T
+								//   then for the Ith local edit on the branch we compose...
+								//     - the inverse of the local edit: 1
+								//     - the previous composition: 1
+								//     - the rebased version of the local edit: 1
+								//   This adds up to 3 edits composed per edit on the branch, except for the last one which is not needed.
+								// Summing for all P edit, this gives us: T + 3P - 3.
+								// As part of rebasing the local onto the tip of the trunk,
+								//   For the Ith peer edit, we compose...
+								//     - the peer edit to generate the changeset: 1
+								//   Summing over all P edits transforms it into P.
```

Comment:
With op bunching turned on, all the peer edits will be processed in a single compose call so I think doesn't make sense to describe the computation per peer edit
```suggestion
								// As part of rebasing the local branch onto the tip of the trunk,
								//   we compose all P edits
```